shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/data/remote/datasource/SurveyRemoteDataSource.kt

```diff
@@ -9,11 +9,17 @@ import kotlinx.coroutines.flow.Flow
 
 interface SurveyRemoteDataSource {
     fun getSurveys(params: GetSurveysQueryParam): Flow<List<SurveyApiModel>>
+
+    fun getSurvey(id: String): Flow<SurveyApiModel>
 }
 
 class SurveyRemoteDataSourceImpl(private val apiClient: ApiClient) : SurveyRemoteDataSource {
 
     override fun getSurveys(params: GetSurveysQueryParam): Flow<List<SurveyApiModel>> {
         return apiClient.responseBodyWithParams(Endpoint.SURVEYS, HttpMethod.Get, params)
     }
+
+    override fun getSurvey(id: String): Flow<SurveyApiModel> {
+        return apiClient.responseBody(Endpoint.SURVEYS + "/${id}", HttpMethod.Get)
```

Comment:
We can omit the `{}` here.
```suggestion
        return apiClient.responseBody(Endpoint.SURVEYS + "/$id", HttpMethod.Get)
```

Reply:
solved in [05a48e0](https://github.com/nimblehq/avishek-phong-kmm-ic/pull/119/commits/05a48e0eab1776cafd561a4724b5775f2122792c)